tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -338,10 +336,6 @@ def _a11y_click_menu_item(self, menu_item_locator):
         self.accessibility.click(menu_item)
         self._wait_for_parent_section_not_displayed(menu_item)
 
-    def _wait_for_toggle_ready(self, by, locator):
```

Comment:
This is a method that's already not being used, see: http://mxr.mozilla.org/gaia/search?string=_wait_for_toggle_ready